src/components/order-summary/_order-summary.scss

```diff
@@ -0,0 +1,147 @@
+.bx--order-summary {
+  @include font-size('14');
+  background-color: $ui-01;
+  padding-bottom: 1.25rem;
+  width: 18rem;
+  height: 100%;
+  min-height: 100vh;
```

Comment:
Wasn't sure the context in which this might be used (will there be a header?), so I've set it to a min-height `100vh` as of now. Should this be changed to a different value?